toolkit/recordreplay/JSControl.cpp

```diff
@@ -616,7 +622,7 @@ static bool Method_OnHttpRequest(JSContext* aCx, unsigned aArgc, Value* aVp) {
   nsAutoCString requestId;
   ConvertJSStringToCString(aCx, args.get(0).toString(), requestId);
 
-  double bookmark = args.get(1).toDouble();
+  double bookmark = args.get(1).toNumber();
```

Comment:
Turns out this was wrong and I was just lucky before!

Reply:
Yeah, toNumber() is best to use in spidermonkey to handle both the int32 and double cases.